prepare-cross-toolchain.sh

```diff
@@ -52,14 +52,19 @@ cp $SRC/$CROSS_ARCH-w64-mingw32/bin/*.dll $DEST/bin
 rm -rf $DEST/lib/clang/$CLANG_VERSION
 cp -a $CLANG_RESOURCE_DIR $DEST/lib/clang/$CLANG_VERSION
 
-rm -rf $DEST/include
-# Copy over headers and arch specific files, converting a unix style
-# install (everything in arch specific subdirectories) into
-# what we'd have when built on Windows, as if build-mingw-w64.sh
-# was called with --skip-include-triplet-prefit, with all headers
-# in $DEST/include, and only keeping the bin and lib directories for the
-# individual architectures.
-cp -a $SRC/generic-w64-mingw32/include $DEST/include
+if [ -z "$FULL_LLVM" ]; then
```

Comment:
If we've passed `--full-llvm` to `build-cross-tools.sh`, then we do set `FULL_LLVM` as a variable within that script, but we don't export it, so I'm not sure if the value actually does get propagated here? Either we'd need to export the variable there, or pass `--full-llvm` onto this script as well (like we do for `build-llvm.sh` in that script) and add argument parsing here.

But I wonder if we really need the `rm -rf $DEST/include` here at all. I'll check what contents there is in that directory after the `strip-llvm.sh` script has executed (I'm running a test build, which takes a little while); it may very well be that we don't need it at all (and I just kept it due to the form of the `cp` command I used before). In that case we could just ditch the `rm -rf` and change the `cp` command to your suggested form, and don't need to pass the parameter here at all.

Reply:
Ah, I only tested with the Docker based `release.sh` workflow where `FULL_LLVM` gets set via `--build-arg` passed to the docker command and in there is used via `ARG` which exports it, so I could just use it.

If your build succeeds we can just use your suggestion and be done with it :-)

Reply:
It's indeed empty at this point, so we should be able to replace the `rm -rf` with a `mkdir -p`, to make sure it exists, and then change the `cp` command as you suggest, to merge its contents with whatever is there from before.

Reply:
I've adjusted the commit and force-pushed it with your suggestion.